superset-frontend/plugins/plugin-chart-word-cloud/src/chart/WordCloud.tsx

```diff
@@ -158,6 +158,7 @@ class WordCloud extends React.PureComponent<
   update() {
     const { data, encoding } = this.props;
 
+    // @ts-ignore
```

Comment:
not a huge deal, but I'd be curious to understand what the error was here, do you have it handy?

Reply:
Sure, its

```
Argument of type 'Partial<DeriveEncoding<WordCloudEncodingConfig>>' is not assignable to parameter of type 'never'
```

Reply:
Oh, I just noticed it's still a class component. Let's revisit this some day when it's migrated to a functional component.